aladdin.sh

```diff
@@ -123,26 +123,38 @@ function enter_minikube_env() {
 }
 
 function _extract_cluster_config_value() {
+    # Try extracting config from cluster config.json, default config.json, then aladdin config.json
     local value
     value="$1"
-    jq -r ".$value" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json"
+    jq -nr --arg value "$value" "first(inputs | .$value // empty)" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json" \
```

Comment:
Discussed in PM but for posterity: needs single quotes around `first(....)` and square brackets around `$value`